README.md

```diff
@@ -1,14 +1,14 @@
 # WebDriver BiDi
 
-WebDriver BiDi is a proposed bidirectional protocol for browser automation,
+WebDriver BiDi is a bidirectional protocol for browser automation,
 building on and extending [WebDriver](https://w3c.github.io/webdriver/).
 
-WebDriver BiDi is not ready. Here's what we have so far:
+For more info, consult these resources:
```

Comment:
Should we say that it is a living spec and continuously gets new features added?